orderbook/src/database/orders.rs

```diff
@@ -48,28 +48,54 @@ impl DbOrderKind {
     }
 }
 
+#[derive(sqlx::Type)]
+#[sqlx(rename = "SigningScheme")]
+#[sqlx(rename_all = "lowercase")]
+pub enum DbSigningScheme {
+    EIP712,
+}
+
+impl DbSigningScheme {
+    pub fn from(signing_scheme: SigningScheme) -> Self {
+        match signing_scheme {
+            SigningScheme::EIP712 => Self::EIP712,
+        }
+    }
+
+    fn into(self) -> SigningScheme {
+        match self {
+            Self::EIP712 => SigningScheme::EIP712,
+        }
+    }
+}
+
 impl Database {
     pub async fn insert_order(&self, order: &Order) -> Result<(), InsertionError> {
         const QUERY: &str = "\
             INSERT INTO orders (
-                uid, owner, creation_timestamp, sell_token, buy_token, sell_amount, buy_amount, \
-                valid_to, app_data, fee_amount, kind, partially_fillable, signature) \
-            VALUES ( \
-                $1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13);";
+                uid, owner, creation_timestamp, sell_token, buy_token, receiver, sell_amount, buy_amount, \
+                valid_to, app_data, fee_amount, kind, partially_fillable, signature, signing_scheme) \
```

Comment:
Nit: If each of these were on a different line it would be easier to detect which new components had been added or removed.

Reply:
Agree it would be easier to detect but also annoying to read because it would be so many lines. Not sure what is better.